data/saas/config/hubspot_config.yml

```diff
@@ -56,18 +56,13 @@ saas_config:
               source: body
               path: paging.next.link
         update:
-          path: /crm/v3/objects/contacts/<contactId>
-          method: PATCH
-          body: |
-            {
-              <masked_object_fields>
-            }
           param_values:
             - name: contactId
               references:
                 - dataset: <instance_fides_key>
                   field: contacts.id
                   direction: from
+          request_override: hubspot_contacts_update
```

Comment:
This isn't strictly enforced but we usually put the `request_override` before `param_values`

Reply:
It _felt_ a bit wonky the way I had it, thanks :)